experimental/bridge/opentracing/wrapper.go

```diff
@@ -0,0 +1,71 @@
+package opentracing
+
+import (
+	"context"
+
+	ot "github.com/opentracing/opentracing-go"
+
+	otelcore "go.opentelemetry.io/api/core"
+	oteltrace "go.opentelemetry.io/api/trace"
+)
+
+type WrapperTracer struct {
```

Comment:
Please comment on how the WrapperTracer type functions.

Reply:
Will do.